.github/workflows/CI.yml

```diff
@@ -12,26 +12,21 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.0'  # Oldest version supported by this package
+          - '1.6'  # Oldest version supported by this package
```

Comment:
This doesn't match
https://github.com/JuliaDSP/DSP.jl/blob/d85a9ebbc6bf5fdc58486955f8bdac7405fda215/Project.toml#L25
and I think that if we claim to support Julia down to 1.0, we should test against it.
But I guess it's quite reasonable to actually bump the compat to `1.6`.

Reply:
I think 1.6 should be reasonable.